flake.nix

```diff
@@ -13,15 +13,27 @@
   inputs = {
     # nixpkgs.url = "nixpkgs/nixos-unstable";
     nixpkgs.url = "nixpkgs/22.05";
-    flake-compat.url = "github:edolstra/flake-compat";
-    flake-compat.flake = false;
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
     flake-utils.url = "github:numtide/flake-utils";
-    pre-commit-hooks.url = "github:cachix/pre-commit-hooks.nix";
-    pre-commit-hooks.inputs.nixpkgs.follows = "nixpkgs";
-    pre-commit-hooks.inputs.flake-utils.follows = "flake-utils";
+    pre-commit-hooks = {
+      url = "github:cachix/pre-commit-hooks.nix";
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+        flake-utils.follows = "flake-utils";
+      };
+    };
     haskell-nix-utils.url = "github:TerrorJack/haskell-nix-utils";
-    jupyterWith.url = "github:tweag/jupyterWith";
-    # jupyterWith.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
@idontgetoutmuch do you remember why this was commented? Is there harm in keeping this in sync with our nixpkgs input?

Reply:
Ah, I guess because it simply doesn't build like that.

Reply:
This is probably https://github.com/tweag/monad-bayes/pull/231

Reply:
Let's see whether it works after a rebase.